main/urls.py

```diff
@@ -154,6 +154,7 @@
 router.register(r'dref-op-update', dref_views.DrefOperationalUpdateViewSet, basename='dref_operational_update')
 router.register(r'dref-final-report', dref_views.DrefFinalReportViewSet, basename='dref_final_report')
 router.register(r'review-country', api_views.CountryOfFieldReportToReviewViewset, basename='review_country')
+router.register(r'erutype', deployment_views.ERUTypeViewset, basename='erutype')
```

Comment:
We might not want to use the `router.register` here but instead just map a url to a view like https://github.com/IFRCGo/go-api/blob/develop/main/urls.py#L166 .. let's coordinate on teams and can sketch out a basic structure. cc @thenav56